session/determine.go

```diff
@@ -2,15 +2,17 @@ package session
 
 import (
 	"log"
+
+	"github.com/joshmedeski/sesh/config"
 )
 
-func Determine(choice string) Session {
+func Determine(choice string, config *config.Config) Session {
```

Comment:
If there's a default config should this be required and not optional?

Reply:
I'm not sure I follow - Go doesn't really have optional parameters. What you're seeing here is that the Determine function accepts a pointer to a config.Config object to avoid creating copies of the object every time it's drilled down.

Reply:
Reading your comment again, and I think I just misunderstood your question. I think the reason to always pass the config struct through is because the `ParseConfigFile` function will return a struct with all of the default values, and I imagine that will be useful for conditional logic throughout the various commands.